easybuild/easyblocks/c/cp2k.py

```diff
@@ -94,7 +95,7 @@ def extra_options():
             'extradflags': ['', "Extra DFLAGS to be added", CUSTOM],
             'ignore_regtest_fails': [False, ("Ignore failures in regression test "
                                              "(should be used with care)"), CUSTOM],
-            'maxtasks': [3, ("Maximum number of CP2K instances run at "
+            'maxtasks': [4, ("Maximum number of CP2K instances run at "
```

Comment:
this is for the hybrid build. It must be a multiple of two: each process gets two threads.